packages/react/src/Form/core/getComponentClassName.ts

```diff
@@ -8,7 +8,7 @@ const getLastClassName = (classNameToUse: string) => {
   return classNameToUse
     .split(" ")
     .reverse()
-    .find((it) => !!it);
+    .find((it) => Boolean(it));
```

Comment:
On ne peut pas juste faire ça ?
```suggestion
    .find(Boolean);
```

Reply:
En effet ça a l'air de fonctionner 👍